docs/divide-style.md

```diff
@@ -10,27 +10,27 @@ This functionality is not yet supported! If you need this, reach out to us on [#
 
 ## Quick reference
 
-| Class                   | Description           |
-|-------------------------|-----------------------|
-| `divide-solid` > * + *  | border-style: solid;  |
-| `divide-dashed` > * + * | border-style: dashed; |
-| `divide-dotted` > * + * | border-style: dotted; |
-| `divide-none` > * + *   | border-style: none;   |
+| Class                       | Description           |
+|-----------------------------|-----------------------|
+| ~~`divide-solid`~~ > * + *  | border-style: solid;  |
+| ~~`divide-dashed`~~ > * + * | border-style: dashed; |
+| ~~`divide-dotted`~~ > * + * | border-style: dotted; |
+| ~~`divide-none`~~ > * + *   | border-style: none;   |
 
 ## Basic usage
 
 ### Set the divide style
 Control the border style between elements using the `divide-{style}` utilities.
 
-<container>
+<!--container>
```

Comment:
Why keep these commented out?

Reply:
Making it easier to add them back when/if we implement it.